### PR TITLE
#Feature-5 Added-Badge-Component

### PR DESCRIPTION
**Pull request by sushmita12345**
Badge component added to component library.

Sample
Screenshot 2022-02-02 at 8 51 43 PM

### CODE REVIEW

**Review by PuneetP16 — APPROVED** [review]

**Comment by j24m** [comment]
The badges on the icons are way too small, you can change them a bit. rest is good.

**Review by j24m — COMMENTED** [review]
Color variable names can be changed , like white can be --light-color instead of white or red can be --dark-color or --red-shade-color. Rest is good to merge, clean code and nice UI.

**Comment by deekshithmd** [comment]
Badges looks small for icons and work on badges placement

File: assets/css/badge.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+.badge-green{
+    height: 2.2vh;
+    width: .5vw;
+    border-radius: 50%;
+    border: .2rem solid var(--status-badge);
+    background-color: var(--green);
+    position: absolute;
+    top: 8.5rem;
```
Comment (deekshithmd) [inline assets/css/badge.css:27]: using left  makes badge fixed at that place irrespective of image size

Comment (sushmita12345) [reply assets/css/badge.css:27]: Thanks, changes has been done.

**Review by palash23jain — APPROVED** [review]

**Pull request merged**